### PR TITLE
Update for iosas 1.02a

### PR DESCRIPTION
**Pull request by JeodC**
Update this port to use the latest Steam and Itch.io game version, 1.02a. Tested on RG552.

### CODE REVIEW

**Comment by JeodC** [comment]
This PR requires the love patcher that is currently on the gui beta branch.

**Review by Slayer366 — APPROVED** [review]

**Review by kloptops — DISMISSED** [review]

File: ports/iosas/Isles of Sea and Sky.sh [hunk 1]
```diff
@@ -13,106 +13,56 @@ else
 fi
 
 source $controlfolder/control.txt
-source $controlfolder/device_info.txt
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:16]: can be removed

File: ports/iosas/Isles of Sea and Sky.sh [hunk 2]
```diff
@@ -13,106 +13,56 @@ else
...
 [ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 get_controls
 
-# Setup permissions
-$ESUDO chmod 666 /dev/tty1
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:21]: can be removed

File: ports/iosas/Isles of Sea and Sky.sh [hunk 3]
```diff
@@ -13,106 +13,56 @@ else
...
-$ESUDO chmod 666 /dev/uinput
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:22]: can be removed

File: ports/iosas/Isles of Sea and Sky.sh [hunk 4]
```diff
@@ -13,106 +13,56 @@ else
...
-$ESUDO chmod 777 "$GAMEDIR/tools/gm-Ktool.py"
-
 # Exports
 export LD_LIBRARY_PATH="$GAMEDIR/libs:$GAMEDIR/tools/lib:$LD_LIBRARY_PATH"
 export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
-export PATH="$GAMEDIR/tools:$PATH"
-
-# Display loading splash
-if [ -f "$GAMEDIR/game.droid" ]; then
-    $ESUDO ./libs/splash "splash.png" 1
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:43]: As it seems to only be needed in muOS maybe you can do this
```
[ "$CFW_NAME" == "muOS" ] && splash "splash.png" 1 # workaround for muOS
```

File: ports/iosas/Isles of Sea and Sky.sh [hunk 5]
```diff
@@ -13,106 +13,56 @@ else
...
 
 # Assign gptokeyb and load the game
 $GPTOKEYB "gmloadernext" -c "control.gptk" &
+pm_platform_helper "$GAMEDIR/game.apk"
 ./gmloadernext game.apk
 
 # Kill processes
-$ESUDO kill -9 $(pidof gptokeyb)
+pm_finish
 $ESUDO systemctl restart oga_events &
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:67]: can be removed

File: ports/iosas/Isles of Sea and Sky.sh [hunk 6]
```diff
@@ -13,106 +13,56 @@ else
...
 printf "\033c" > /dev/tty0
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:68]: can be removed

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/iosas/Isles of Sea and Sky.sh [hunk 7]
```diff
@@ -13,106 +13,54 @@ else
...
-    sed -i 's/malgun\.ttf/BMDOHYEON_ttf.ttf/g' localization_fonts.csv # Korean
+# Display loading splash
+if [ -f "$GAMEDIR/patchlog.txt" ]; then
+    [ "$CFW_NAME" == "muOS" ] && splash "splash.png" 1 # workaround for muOS
+    $ESUDO ./libs/splash "splash.png" 8000
 fi
 
 # Assign gptokeyb and load the game
 $GPTOKEYB "gmloadernext" -c "control.gptk" &
+pm_platform_helper "$GAMEDIR/gmloadernext.apk"
```
Comment (cdeletre) [inline ports/iosas/Isles of Sea and Sky.sh:62]: Almost ;)

**Review by cdeletre — CHANGES_REQUESTED** [review]

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**